tests/test_util_modules.py

```diff
@@ -452,6 +458,8 @@ def test_split_parts():
 
 
 def test_generic_data_classes():
+    from hivemind.utils import ValueWithExpiration, HeapEntry, DHTExpiration
```

Comment:
Can we return these imports back to the beginning of the file?

Reply:
thx, fixed